### PR TITLE
Update Dockerfile for Stretch

### PR DESCRIPTION
**Pull request by mdholloway**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,11 +1,9 @@
...
+FROM debian:stretch
+RUN apt-get update && apt-get install -y nodejs nodejs-legacy git wget python libcairo2-dev libjpeg62-turbo-dev libpango1.0-dev libgif-dev build-essential g++ libmapnik-dev && rm -rf /var/lib/apt/lists/*
 ENV NVM_DIR /usr/local/nvm
 RUN wget -qO- https://raw.githubusercontent.com/creationix/nvm/v0.33.6/install.sh | bash && . $NVM_DIR/nvm.sh && nvm install 6.11.1
 ENV HOME=/root/ LINK=g++
 ENV IN_DOCKER=1
 COPY . /home/code
 WORKDIR /home/code
-CMD . $NVM_DIR/nvm.sh && nvm use 6.11.1 && npm install --build-from-source=mapnik --fallback-to-build=false && npm test
+CMD . $NVM_DIR/nvm.sh && nvm use 6.11.1 && npm install && npm test
```
Comment (pnorman) [inline Dockerfile]: Won't this now pull the binary blob for Mapnik?

Comment (mdholloway) [reply Dockerfile]: Yes, that was what I assumed was happening with these flags removed.  It builds and the tests pass in this case, which suggests to me that possibility (1) from https://phabricator.wikimedia.org/T188674 (using binary blobs in prod) is viable now, on Stretch, if we want to go that route.  But I'll play with it some more while keeping the flags in place.

**Comment by mdholloway** [comment]
@pnorman What is the downside to using binary blobs that we're trying to avoid?

File: Dockerfile (hunk 1 above)
Comment (mdholloway) [reply Dockerfile]: Backed out this part of the change.

**Comment by pnorman** [comment]
They tend to have more problems, which is why we moved to them in the first place. There are also the problems with using static libraries, we don't know what's going into them, and they can't be reproduced.

**Review by thesocialdev — CHANGES_REQUESTED** [review]
It needs one last change at https://github.com/kartotherian/kartotherian/blob/master/.travis.yml#L6

The command should be `docker build --no-cache -t kartotherian .`, otherwise it will use cache for the last step everytime.

We could also use some workarounds to disable cache on the last command only, like this https://stackoverflow.com/questions/35134713/disable-cache-for-specific-run-commands. Which is even better.

I tested the solution above and it works.

**Review by thesocialdev — APPROVED** [review]

**Pull request merged**